tools/rest_api.rb

```diff
@@ -1,293 +1,6 @@
-#!/usr/bin/env ruby
-#
-# Helper Script to access the ManageIQ REST API
-#
-# Makes use of the Trollop and Faraday Client Gems
-#   gem install trollop
-#   gem install faraday
-#   gem install faraday_middleware
-#
-# Run without options to see the usage.
-#
-# Globals accessible by loaded test scripts
-#
+#! /usr/bin/env ruby
+require 'bundler/setup'
+require 'pathname'
 
-require 'uri'
-require 'json'
-require 'trollop'
-require 'faraday'
-require 'faraday_middleware'
+load Pathname.new(Gem::Specification.find_by_name('manageiq-api').gem_dir).join('exe', 'manageiq-api')
```

Comment:
Interested to see how it works out.  The previous implementation is definitely a more of a public API than what I suggested, but this might make sure we are going through bundler.

Anyway, was mostly a suggestion, and if it causes problems, feel free to use the old code.